virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc

```diff
@@ -5,28 +5,21 @@ include::modules/virt-document-attributes.adoc[]
 
 toc::[]
 
-A _boot source_ contains a bootable operating system (OS) and all of the configuration settings for the OS, such as drivers.
+include::modules/virt-about-vms-and-boot-sources.adoc[leveloffset=+1]
 
-You use a boot source to create virtual machine templates with specific configurations. These templates can be used to create any number of available virtual machines.
-
-.Prerequisites
-* To add a boot source, you must be logged in as a user with the `os-images.kubevirt.io:edit` RBAC role or as an administrator. You do not need special privileges to create a virtual machine from a template with a boot source attached.
-
-[NOTE]
-====
-For information about requirements for Microsoft Windows, refer to link:https://www.openshift.com/blog/creating-an-operating-system-image-using-the-microsoft-windows-installation-media-with-openshift-virtualization[Creating installation media for Windows boot sources].
-====
+include::modules/virt-importing-rhel-image-boot-source-web.adoc[leveloffset=+1]
 
-// Commenting this module with list of supported DV names for 2.6. It is relevant only to 2.6.1 when the CLI flow is introduced
-// include::modules/virt-using-supported-boot-source-names.adoc[leveloffset=+1]
+// Placeholder for module CNV-9002 Importing (Preparing?) a custom boot source image on Linux
 
 include::modules/virt-adding-a-boot-source-web.adoc[leveloffset=+1]
 
 include::modules/virt-creating-a-vm-from-a-template-with-an-attached-boot-source.adoc[leveloffset=+1]
 
-include::modules/virt-importing-rhel-image-boot-source-web.adoc[leveloffset=+1]
-
+[id="additional-resources_creating-and-using-boot-sources"]
 == Additional resources
-* xref:../../../virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-web.adoc#virt-uploading-local-disk-images-web[Uploading local disk images]
-
-* xref:../../../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[Creating virtual machines]
+* xref:../../../virt/vm_templates/virt-creating-vm-template.adoc#virt-create-vms[Creating virtual machine templates]
+* link:https://www.openshift.com/blog/creating-windows-virtual-machines-from-existing-images-with-openshift-virtualization[Creating a PVC with a Microsoft Windows boot source]
+* link:https://www.openshift.com/blog/customizing-virtual-machine-templates-in-openshift-1[Customizing virtual machine templates for Microsoft Windows in {product-title}]
+* link:https://www.openshift.com/blog/creating-default-os-images-to-auto-clone-in-openshift-virtualization[Setting a PVC as a boot source for a Microsoft Windows template]
```

Comment:
@bgaydosrh I think we should add a word about this blog using command-line, and not the gui-based method that is new to 4.8.

Reply:
@cwilkers @jobbler can you think about better one-liners to describe these blog posts?

Reply:
@dankenigsberg - Sure, let me know when the names are finalized. I would like to get this merged by early next week. We can add the command-line info in the title for the blog you refer to. We are not supposed to display anything but titles in this section (no descriptions --- the titles should be descriptive, but don't have to match the blog title precisely).

Reply:
Changed title to: _Setting a PVC as a boot source for a Microsoft Windows template using the CLI_